src/components/Header/Header.tsx

```diff
@@ -0,0 +1,26 @@
+type Props = {
+  title: string,
```

Comment:
The name of variable is a little bit confusing as it's not a title but a value of an input. Maybe something like todoInputValue serves better